### PR TITLE
views/tx: show count of inputs and outputs

### PR DESCRIPTION
**Pull request by McEdward**
screenshot 126

### CODE REVIEW

**Comment by McEdward** [comment]
resolving issue #824

**Comment by raedah** [comment]
Count column doesn't add any value. Sum at the top is all thats needed.

**Comment by chappjc** [comment]
Let's consider this as a proposal until more people have had the opportunity to register their opinion on the numbering of each i/o row (as shown).  As I mentioned in issue #241 , the best way to convey this row-wise information is not clear.  This PR is one way that does eat a lot of horizontal space, and there may be tighter ways of conveying this info, even if that means it is not displayed except by mouse hover, etc.

**Comment by buck54321** [comment]
Either but not both seems fine. Also, it's technically not the row count, but the vin/vout index, so it does have meaning beyond simply enumerating the rows.

**Comment by chappjc** [comment]
Agreed, it's good to have.  When an output links to tx input where the output is spent, that index needs to be visible somewhere on the spending transaction's view as this PR does.
We can move forward with this and see what ideas pop out of the woodwork.

File: explorer/explorertypes.go [hunk 1]
```diff
@@ -129,6 +129,8 @@ type TxInfo struct {
 	Type             string
 	Vin              []Vin
 	Vout             []Vout
+	VinLenght        int
```
Comment (buck54321) [inline explorer/explorertypes.go]: Lenght -> Length

**Comment by chappjc** [comment]
There is also the possibility of giving the user the ability to select which columns to display.
It could be bulky and horrible, but if done in a modern and unobtrusive way, it could be pretty slick.

**Comment by raedah** [comment]
I originally just thought it was row numbers that happened to start at zero, and that seemed a bit useless. But once I got that it was the vin/vout index I saw that it had meaning. Could put a help hover over on the column header which says 'vin/vout index'. 

Also seeing that instead of 
Input (1)    Outputs (3)
could just be
1 Input    3 Outputs

**Comment by gozart1** [comment]
We could add verbs to help make the mechanics of a transaction a bit more explicit.

1 Input Consumed
2 Outputs Created

Inspired by Block Cypher, eg
https://live.blockcypher.com/btc/tx/0bde300106882ff2b98081419aa7465a64c00fbc4c2d573bf6593da8dc4cb489

**Comment by chappjc** [comment]
@gozart1 I like that consumed/created language.

To be explicit about the value of the in/out index numbering, when discussing a transaction with more than a handful of inputs/output, it is hard to say which input or output you want to direct someone's attention to without having the index shown.

I also like the idea of putting hover text (`title`) for the column headers.

@McEdward Would you mind demoing the above changes?

**Pull request merged**